.github/workflows/main.yml

```diff
@@ -157,7 +157,7 @@ jobs:
           /bin/bash ci/build_naive.sh
 
   router:
-    runs-on: [ self-hosted, linux, light ]
+    runs-on: [ self-hosted, linux, k8s-runner-no-compose ]
```

Comment:
let's leave basic jobs as is for now

Reply:
that`s working absolutely correct 
 https://github.com/datalens-tech/datalens-backend/actions/runs/11107711169